src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -296,11 +330,339 @@ const formConfig = {
+                  }),
+                  { 'ui:description': additionalDocumentDescription }
+                )
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {
+                    additionalDocuments: {
+                      type: 'array',
+                      items: {
+                        type: 'object',
+                        required: ['name', 'attachmentId'],
+                        properties: {
+                          name: {
+                            type: 'string'
+                          },
+                          size: {
+                            type: 'integer'
+                          },
+                          confirmationCode: {
+                            type: 'string'
+                          },
+                          attachmentId: {
+                            type: 'string',
+                            'enum': [
+                              '1',
+                              '2',
+                              '3',
+                              // '4',
+                              '5',
+                              '6'
+                            ],
+                            enumNames: [
+                              'Discharge',
+                              'Marriage related',
+                              'Dependent related',
+                              // 'VA preneed form',
+                              'Letter',
+                              'Other'
+                            ]
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        },
+        evidenceSummary: {
+          title: 'Summary of evidence',
+          depends: () => {
+            // TODO: needs one of the types of evidence?, checkbox validation?
+            // const hasOtherEvidence = _.get(`disabilities.${index}.view:otherEvidence`, formData);
+            // return hasOtherEvidence;
+            return true;
+          },
+          path: 'supporting-evidence/:index/evidence-summary',
+          // editPageOnReview: true,
+          showPagePerItem: true,
+          arrayPath: 'disabilities',
+          uiSchema: {
+            disabilities: {
+              items: {
+                'ui:title': 'Summary of evidence',
+                'ui:description': evidenceSummaryView
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              disabilities: {
+                type: 'array',
+                items: {
+                  type: 'object',
+                  properties: {}
+                }
+              }
+            }
+          }
+        }
```

Comment:
Kindly focus on the `evidenceSummary` page, as that is the updated portion of this file in this PR, other changes refer to other PRs that this is branched off of (and will not be displayed here once those are merged in).